cli/asc.js

```diff
@@ -342,7 +346,62 @@ exports.main = function main(argv, options, callback) {
           }
         }
       }
-      if (sourceText == null) {
+      /*
+      In this case the library wasn't found so we check paths
+      */
+      if (sourceText == null && args.path) {
+        writeStdout(`Looking for ${sourcePath}\n`)
```

Comment:
Should go to stderr - stdout might be module output

Reply:
Okay will change.

Reply:
I guess that should also be the case for `traceResolution`.

Reply:
In general I think that the debugging might help now to implement the feature, but I'm skeptical that it deserves actual compiler options that nobody will ever use, unless debugging the feature.

Reply:
Well typescript has the option and I did need a way to debug it.  I'm not attached though.  `tsc` also has `--all` which is where you'll find these types of commands not just `--help`.

Reply:
Hmm, hiding these behind an `-all` flag sounds good, yeah.

Reply:
Should I add that now or do a future PR?